src/pubd/repository.rs

```diff
@@ -1285,7 +1302,7 @@ impl RepoStats {
     pub fn stale_publishers(&self, seconds: i64) -> Vec<PublisherHandle> {
         let mut res = vec![];
         for (publisher, stats) in self.publishers.iter() {
-            if let Some(update_time) = stats.last_update {
+            if let Some(update_time) = stats.last_update() {
```

Comment:
This looks like a logic fix unrelated to the PR? Is that correct?

Reply:
Because of the inclusion of server information I needed to change to a dynamically calculated structure. This change is a result of that.